### PR TITLE
[feature] add at function for wechat

### PR DESCRIPTION
**Pull request by Yanshuming1**
Added enterprise weChat robot  @ function

### CODE REVIEW

**Comment by Yanshuming1** [comment]
@tomsun28 cr ，please

File: manager/src/main/java/org/apache/hertzbeat/manager/component/alerter/impl/WeWorkRobotAlertNotifyHandlerImpl.java [hunk 1]
```diff
@@ -71,6 +84,34 @@ public void send(NoticeReceiver receiver, NoticeTemplate noticeTemplate, Alert a
...
     }
 
+    private WeWorkWebHookDto checkNeedAtNominator(NoticeReceiver receiver, Alert alert) {
+        if (StringUtils.isBlank(receiver.getPhone()) && StringUtils.isBlank(receiver.getTgUserId())) {
+            return null;
+        }
+        WeWorkWebHookDto weWorkWebHookTextDto = new WeWorkWebHookDto();
+        weWorkWebHookTextDto.setMsgtype(WeWorkWebHookDto.TEXT);
+        WeWorkWebHookDto.TextDTO textDto = new WeWorkWebHookDto.TextDTO();
+        String alertMessage = String.format("警告对象：%s\n详情：%s", alert.getTarget(), alert.getContent());
```
Comment (crossoverJie) [inline manager/src/main/java/org/apache/hertzbeat/manager/component/alerter/impl/WeWorkRobotAlertNotifyHandlerImpl.java:94]: Can these be replaced with `i18n`?

File: manager/src/main/java/org/apache/hertzbeat/manager/component/alerter/impl/WeWorkRobotAlertNotifyHandlerImpl.java [hunk 2]
```diff
@@ -113,6 +158,22 @@ private static class MarkdownDTO {
...
+        private static class TextDTO {
+
+            /**
+             * message content
+             */
+            private String content;
+            /**
+             * @ userId
+             */
+            private List<String> mentioned_list;
```
Comment (crossoverJie) [inline manager/src/main/java/org/apache/hertzbeat/manager/component/alerter/impl/WeWorkRobotAlertNotifyHandlerImpl.java:171]: Recommend using camelCase naming.

**Comment by crossoverJie** [comment]
Code checkstyle is failed, please refer to [here](https://github.com/apache/hertzbeat/blob/master/home/docs/community/code-style-and-quality-guide.md#21-configure-checkstyle) and ensure it passes the local test first.

File: manager/src/main/java/org/apache/hertzbeat/manager/component/alerter/impl/WeWorkRobotAlertNotifyHandlerImpl.java [hunk 3]
```diff
@@ -71,6 +82,34 @@ public void send(NoticeReceiver receiver, NoticeTemplate noticeTemplate, Alert a
...
+            return null;
+        }
+        WeWorkWebHookDto weWorkWebHookTextDto = new WeWorkWebHookDto();
+        weWorkWebHookTextDto.setMsgtype(WeWorkWebHookDto.TEXT);
+        WeWorkWebHookDto.TextDTO textDto = new WeWorkWebHookDto.TextDTO();
+        String alertMessage = String.format("警告对象：%s\n详情：%s", alert.getTarget(), alert.getContent());
+        textDto.setContent(alertMessage);
+        if (StringUtils.isNotBlank(receiver.getPhone())) {
+            textDto.setMentioned_mobile_list(analysisArgToList(receiver.getPhone()));
+            weWorkWebHookTextDto.setText(textDto);
```
Comment (gjjjj0101) [inline manager/src/main/java/org/apache/hertzbeat/manager/component/alerter/impl/WeWorkRobotAlertNotifyHandlerImpl.java:96]: maybe line 96 and 100 can combine into one line

Comment (Yanshuming1) [reply manager/src/main/java/org/apache/hertzbeat/manager/component/alerter/impl/WeWorkRobotAlertNotifyHandlerImpl.java:97]: The two entries are different

File: manager/src/main/java/org/apache/hertzbeat/manager/component/alerter/impl/WeWorkRobotAlertNotifyHandlerImpl.java (hunk 1 above)
Comment (Yanshuming1) [reply manager/src/main/java/org/apache/hertzbeat/manager/component/alerter/impl/WeWorkRobotAlertNotifyHandlerImpl.java:93]: At present, we do not know the back-end i8n, and we can modify it after learning

File: web-app/src/assets/i18n/en-US.json [hunk 4]
```diff
@@ -271,6 +271,7 @@
   "alert.notice.type.wechat-id": "WeChat OPENID",
   "alert.notice.type.wework": "WeWork Robot",
   "alert.notice.type.wework-key": "WeWork Robot KEY",
+  "alert.notice.type.wework-userId": "userId",
```
Comment (tomsun28) [inline web-app/src/assets/i18n/en-US.json:274]:
```suggestion
  "alert.notice.type.wework-userId": "UserId",
```

File: web-app/src/assets/i18n/zh-CN.json [hunk 5]
```diff
@@ -271,6 +271,7 @@
   "alert.notice.type.wechat-id": "微信OPENID",
   "alert.notice.type.wework": "企业微信机器人",
   "alert.notice.type.wework-key": "企业微信机器人KEY",
+  "alert.notice.type.wework-userId": "用户id",
```
Comment (tomsun28) [inline web-app/src/assets/i18n/zh-CN.json:274]: hi, the i18n zh-TW.json also need this.

**Review by tomsun28 — APPROVED** [review]
LGTM👍👍

**Pull request merged**